recipes/maxit/meta.yaml

```diff
@@ -6,13 +6,22 @@ package:
   version: {{ version }}
 
 source:
-  url: https://sw-tools.rcsb.org/apps/{{ name }}/{{ name|lower }}-v{{ version }}-prod-src.tar.gz
-  sha256: 658e236c6310cf7e55218a0500f82050ef86074f5d5b3f61d2a161b04a38cc39
-  patches:
-    - platform.sh.patch  # [osx]
+  - url: https://sw-tools.rcsb.org/apps/{{ name }}/{{ name|lower }}-v{{ version }}-prod-src.tar.gz
+    sha256: 658e236c6310cf7e55218a0500f82050ef86074f5d5b3f61d2a161b04a38cc39
+    patches:
+      - platform.sh.patch
+      - generate_assembly_cif_file.C.patch
+      - maxit.C.patch
+      - process_entry.C.patch
+      - make.platform.gnu8.patch  # [linux]
+      - binary.csh.patch  # [linux]
+      - system-util.C.patch  # [linux]
+      - Makefile.patch  # [linux]
+  - path: ./data-binary_linux-x86_64.tar.xz  # [linux]
```

Comment:
I added compressed file of data/binary/* to use it for build on linux.

Compiling and preparing data/binary/* on linux always fails due to memory shortage.